code/modules/mob/living/carbon/human/emote.dm

```diff
@@ -213,66 +213,39 @@
 /datum/emote/living/carbon/human/dab
 	key = "dab"
 	key_third_person = "*dab"
-	restraint_check = TRUE
 
-/datum/emote/living/carbon/human/dab/can_run_emote(mob/user, var/status_check = TRUE)
+/datum/emote/living/carbon/human/dab/can_run_emote(mob/user)
 	var/mob/living/carbon/human/H = user
-	if (iswizard(H))
-		to_chat(user, "<span class='warning'>The Wizard Federation has banned usage of the [key].</span>")
-		return FALSE
 	if(H.has_organ(LIMB_LEFT_ARM) && H.has_organ(LIMB_RIGHT_ARM))
-		if(user.stat > stat_allowed)
-			to_chat(user, "<span class='warning'>You cannot [key] while unconscious.</span>")
-			return FALSE
-		if(restraint_check && (user.restrained() || user.locked_to))
-			to_chat(user, "<span class='warning'>You cannot [key] while restrained.</span>")
-			return FALSE
 		return TRUE
-	else
-		to_chat(user, "<span class='warning'>You cannot [key] without both your arms.</span>")
-		return FALSE
 
-/datum/emote/living/carbon/human/dab/run_emote(mob/user, params, ignore_status = FALSE)
-	if(!(can_run_emote(user, !ignore_status)))
-		return FALSE
+/datum/emote/living/carbon/human/dab/run_emote(mob/user)
 	var/mob/living/carbon/human/H = user
 	if(!istype(H))
 		return
-	if(!(Holiday == APRIL_FOOLS_DAY))
-		//var/confirm = alert("Suffer for your sins.", "Confirm Suicide", "gladly", "ok")
-		//var/confirm = alert("Are you sure you want to do this? Nobody will want to revive you.", "Confirm Suicide", "Yes", "Yes")
-		//var/confirm = alert("Are you sure you want to [key]? This action will cause irreversable brain damage.", "Confirm Suicide", "Yes", "Yes")
-		var/confirm = alert("Are you sure you want to [key]? This action cannot be undone and you will not able to be revived.", "Confirm Suicide", "Yes", "No")
-		if(confirm != "Yes")
-			return
-		H.suiciding = 1
-		H.visible_message("<span class='danger'>[H] holds one arm up and slams \his other arm into \his face! It looks like \he's trying to commit suicide.</span>",)
-		for(var/datum/organ/external/breakthis in H.get_organs(LIMB_LEFT_ARM, LIMB_RIGHT_ARM, LIMB_HEAD))
-			H.apply_damage(50, BRUTE, breakthis)
-			if(!(H.species.anatomy_flags & NO_BONES))
-				breakthis.fracture()
-		H.adjustOxyLoss(max(175 - H.getToxLoss() - H.getFireLoss() - H.getBruteLoss() - H.getOxyLoss(), 0))
-		H.updatehealth()
+	if(iswizard(H))
+		to_chat(user, "<span class='warning'>The Wizard Federation has banned usage of the [key].</span>")
+		return FALSE
+	if(H.stunned || H.restrained())
```

Comment:
Wasn't the intent that you can dab while cuffed but not while unconcious? That isn't what this does.